zebra-test/src/vectors.rs

```diff
@@ -165,4 +202,11 @@ lazy_static! {
     // https://github.com/zcash/zips/blob/master/zip-0243.rst#test-vector-3
     pub static ref ZIP243_3: Vec<u8> = <Vec<u8>>::from_hex("0400008085202f8901a8c685478265f4c14dada651969c45a65e1aeb8cd6791f2f5bb6a1d9952104d9010000006b483045022100a61e5d557568c2ddc1d9b03a7173c6ce7c996c4daecab007ac8f34bee01e6b9702204d38fdc0bcf2728a69fde78462a10fb45a9baa27873e6a5fc45fb5c76764202a01210365ffea3efa3908918a8b8627724af852fc9b86d7375b103ab0543cf418bcaa7ffeffffff02005a6202000000001976a9148132712c3ff19f3a151234616777420a6d7ef22688ac8b959800000000001976a9145453e4698f02a38abdaa521cd1ff2dee6fac187188ac29b0040048b004000000000000000000000000")
         .expect("Transaction bytes are in valid hex representation");
+
+
+    pub static ref SIGHASH: Vec<SighashTest> = {
+        let json_str = include_str!("vectors/sighash.json");
```

Comment:
I'm not particularly concerned about that binary size, but I would like to avoid making Zebra's git repository 3MB larger.

How big is the JSON file when it's compressed? Can we use a compressor that's already in Zebra's transitive dependencies? (Check cargo-tree for gzip, zip, flate, lzma, zstd or something similar.)

You'll need include_bytes! for the compressed data.